setup.py

```diff
@@ -22,7 +22,7 @@
 
 
 requirements = [
-    "numpy>=1.17.4",
+    "numpy>=1.19.2",
```

Comment:
@antalszava how come this was incremented? Does it need to be? Potentially, SF still runs with NumPy 1.17?

Reply:
Hmm, just thought of adjusting it with `requirements.txt`, to have it consistent. Would it be better to leave it as is? Wasn't sure if there was a reason for the difference between the two versions.